src/usr/local/www/includes/functions.inc.php

```diff
@@ -177,7 +177,7 @@ function disk_usage($slice = '/') {
 }
 
 function swap_usage() {
-	exec("/usr/sbin/swapinfo", $swap_info);
+	exec("/usr/sbin/swapinfo | /usr/bin/tail -1", $swap_info);
```

Comment:
This will not work for multiple swap devices. Instead use `swapinfo | tain -n  -1` which will just remove header line

Reply:
@rbgarga, the fix is to show only one swap info . Or the only one present on system (default) or the total of swap used while using more then one.

I think there is no need to show individual usage od each swap.

Reply:
Oh, that makes senes. Thanks!